app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
- [ ] The views in devise for the registration and login must be modified to match the wireframes and your styling.
You can take a look at the wireframes in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) of the project.

- Your Design:

![Screenshot from 2022-12-17 22-20-45](https://user-images.githubusercontent.com/79058364/208264692-32a9e9be-7e93-46da-96a3-71570d5235e1.png)


- Original template:

![image](https://user-images.githubusercontent.com/79058364/208264719-cde14266-91f0-41af-9259-5993f6715d34.png)

Reply:
hey @BenMukebo 
Thanks so much for the review.
But you will need to sign up before your take to the next page which is the login page which requires just the username and password